va/win32/va_win32.c

```diff
@@ -34,15 +34,15 @@
  * which will be selected when provided with an adapter LUID which
  * does not have a registered VA driver
 */
-const char VAAPI_DEFAULT_DRIVER_NAME[] = "vaon12";
+const char VAAPI_DEFAULT_DRIVER_PATH[] = ".\\vaon12_drv_video.dll";
```

Comment:
Note: Probably want this to be ".\vaon12" to try with and without the suffix in va_OpenDriver

Reply:
Do we need the ".\"? Seems like you could just omit that and let the loader search wherever it's going to search.